src/wp-admin/about.php

```diff
@@ -26,11 +26,11 @@
 		<p class="about-text">
 			<?php printf(
 				/* translators: link to "business-focused CMS" article */
-				__( 'Thank you for using ClassicPress, the <a href="%s">business-focused CMS</a>.' ),
-				'https://www.classicpress.net/blog/2018/10/29/classicpress-for-business-professional-organization-websites/'
+				__( 'Thank you for using ClassicPress, the <a href="%s">CMS for Creators</a>.' ),
+				'https://link.classicpress.net/the-cms-for-creators'
```

Comment:
Need to fix this URL at https://github.com/ClassicPress/subdomain-link first.

Reply:
@striebwj This is the missing puzzle piece.

Reply:
Do we have anything we can even point to?

Reply:
@striebwj That link was what we were supposed to link to. We need an article for that on website.

Reply:
I see the PR in subdomain was just pointing to the home page. I think we are okay to merge this, and merge ClassicPress/subdomain-link#3 until we get an article up on the site.